packages/@okta/vuepress-site/docs/guides/token-inline-hook/extend-express/index.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Extend the sample Express app
+---
+
+The following code extends the local sample Node.js Express application to display the results of the Token Inline Hook claim addition. This step is optional. The Token Inline Hook is functional and the results of the implementation are available in the external service logs, as well as the System Log on your Okta org. But this extension is nice to have!
+
+To extend the local sample Node.js Express application, you need to update the `sample-web-server.js` file.
+
+### Update the web server page
+
+1. Navigate to your project folder `samples-nodejs-express-4` and continue to the `common` folder (`samples-nodejs-express-4/common`).
+2. In an editor, open the `sample-web-server.js` page.
+3. Locate the routing function `app.get('/profile'` and add the code, below, after the `const` declarations and before the `res.render` function.
```

Comment:
I was a little confused here - we say "add the code below" but the snippet shows the entire function and I only needed to copy the middle section of the code.  Can we either only show the code to be copied or update the instructions to say to modify the routing function as in the example below?

Reply:
I like including the code on either side of the extension as "markers", but understand that it could be confusing. I was wondering about that myself. I'll update the instructions to say "modify" the function as you suggested.